Formula/swift.rb

```diff
@@ -24,49 +26,151 @@ class Swift < Formula
 
   # Has strict requirements on the minimum version of Xcode
   # https://github.com/apple/swift/tree/swift-#{version}-RELEASE#system-requirements
-  depends_on xcode: ["11.2", :build]
+  depends_on xcode: ["12.2", :build]
+
+  depends_on "python@3.9"
 
+  uses_from_macos "llvm" => :build
+  uses_from_macos "curl"
   uses_from_macos "icu4c"
+  uses_from_macos "libedit"
+  uses_from_macos "libxml2"
+  uses_from_macos "ncurses"
+  uses_from_macos "sqlite"
+  uses_from_macos "zlib"
+
+  on_linux do
+    resource "six" do
+      url "https://files.pythonhosted.org/packages/6b/34/415834bfdafca3c5f451532e8a8d9ba89a21c9743a0c59fbd0205c7f9426/six-1.15.0.tar.gz"
+      sha256 "30639c035cdb23534cd4aa2dd52c3bf48f06e5f4a941509c8bafd8ce11080259"
+    end
```

Comment:
You can just do `depends_on "six" => :build` now, I think.

Reply:
I'll do this in the next push.